lib/view/screen/login_screen.dart

```diff
@@ -128,6 +131,7 @@ class _LoginScreenState extends State<LoginScreen> with WidgetsBindingObserver {
     final bool isLoggingIn =
         Provider.of<MainViewModel>(context, listen: true).loggingIn;
     if (registered) {
+      //_saveCredentialsForAutoLogin(credentialConfig!);
```

Comment:
Let's remove references to this to not confuse people in this case.

If the method isn't used anymore then remove it altogether